scripts/test/client-api-ghci.sh

```diff
@@ -60,7 +60,7 @@ putTextLn ""
 
 EOL
 
-cd educator || :
+# cd educator || :
```

Comment:
Feel free to just remove this line completelly, it became totally unnecessary recently, I just forgot about it.